src/App.jsx

```diff
@@ -14,70 +16,69 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setSelectedGood] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  const stateOfGood = (good) => {
+    const check = selectedGood === good
+      ? setSelectedGood('') : setSelectedGood(good);
 
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+    return check;
+  };
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
+  const checkGoods = selectedGood ? `${selectedGood} is selected` : 'No goods selected';
```

Comment:
let's use more descriptive names:
```suggestion
  const title = selectedGood ? `${selectedGood} is selected` : 'No goods selected';
```